protocol/protocol.go

```diff
@@ -37,22 +37,22 @@ var (
 	ERROR_COMMAND_PARSE = errors.New("Command parse error")
 
 	//Commands declared once for convenience
-	DEL_COMMAND         = []byte{'d', 'e', 'l'}
-	SUBSCRIBE_COMMAND   = []byte{'s', 'u', 'b', 's', 'c', 'r', 'i', 'b', 'e'}
-	UNSUBSCRIBE_COMMAND = []byte{'u', 'n', 's', 'u', 'b', 's', 'c', 'r', 'i', 'b', 'e'}
-	PING_COMMAND        = []byte{'p', 'i', 'n', 'g'}
-	INFO_COMMAND        = []byte{'i', 'n', 'f', 'o'}
-	SHORT_PING_COMMAND  = []byte{'P', 'I', 'N', 'G'}
-	SELECT_COMMAND      = []byte{'s', 'e', 'l', 'e', 'c', 't'}
-	QUIT_COMMAND        = []byte{'q', 'u', 'i', 't'}
+	DEL_COMMAND         = []byte("del")
+	SUBSCRIBE_COMMAND   = []byte("subscribe")
+	UNSUBSCRIBE_COMMAND = []byte("unsubscribe")
+	PING_COMMAND        = []byte("ping")
+	INFO_COMMAND        = []byte("info")
+	SHORT_PING_COMMAND  = []byte("PING")
+	SELECT_COMMAND      = []byte("select")
+	QUIT_COMMAND        = []byte("quit")
```

Comment:
Comparing an objdump of the two ways of doing it:
```diff
$ diff objdump.txt objdump2.txt
16743c16743
<       mprof.go:144    0x137a0 48c70424f8f81500                MOVQ $type.*+199672(SB), 0(SP)
---
>       mprof.go:144    0x137a0 48c70424f8f81500                MOVQ $runtime.rodata+199672(SB), 0(SP)
39452c39452
<       malloc.c:219    0x2aaf7 4805ffff1300            ADDQ $type.*+70399(SB), AX
---
>       malloc.c:219    0x2aaf7 4805ffff1300            ADDQ $runtime.rodata+70399(SB), AX
```

I think from this we can conclude that there isn't a performance difference between the two ways of declaring the byte arrays.